### ISSUE TITLE
Encoder breaks on AS4100000000

### ISSUE THREAD

**Issue by fjahr** [sipa/asmap#14]
I don't think it is a serious issue but with the current configuration of the encoder breaks on an announcement from AS4100000000 when using it for routeviews/BGP dump data. While that ASN is in the range of normal use this is most likely a route leak.

I posted a [question about this ASN on network engineering StackExchange](https://networkengineering.stackexchange.com/questions/84362/who-what-is-as4100000000) to see if there is some legitimate use behind it that we need to care about.

**Comment by sipa** [comment]
The binary format only permits AS numbers between 1 and 33521664, inclusive. If we ever need bigger numbers, we'll need to define a new format.

**Comment by fjahr** [comment]
Yeah, I saw that. I will try to figure out if this is a problem or if we can safely ignore ASNs >33521664 for now but I think it should be fine from what I can tell so far.
